### PR TITLE
Fix devices tab table scroll #2860

### LINKED ISSUE #2860: Fix devices tab table scroll
**Actual Behaviour**

Currently, devices tab in admin has a scroll

**Expected Behaviour**

Remove scroll

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2860

Changes: Multi-line break for devices table in admin

Demo Link: https://pr-2864-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Screenshot 2019-08-16 at 1 19 17 PM

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks. The user cannot see that the geolocation is a link. Could you please use the link design for the geolocation?

**Comment by mariobehling** [comment]
* For "Date Added" and "Last Login" could you show the time, please?
* For the map show two lines, please (and reduce the width a bit) 
  * Line 1: Lat
  * Line 2: Long

**Comment by fragm3** [comment]
@mariobehling made the changes.

File: src/components/Admin/ListDevices/index.js [hunk 1]
```diff
@@ -41,30 +42,32 @@ class ListDevices extends React.Component {
...
             if (device.geolocation) {
               location = (
                 <span
                   onClick={this.handleClick}
                   name={macId}
-                  style={{ cursor: 'pointer' }}
+                  style={{ cursor: 'pointer', color: '#4285f4' }}
                 >
-                  {device.geolocation.latitude},{device.geolocation.longitude}
+                  {device.geolocation.latitude}
```
Comment (akshatnitd) [inline src/components/Admin/ListDevices/index.js]: Can we add a `,` here?

Comment (akshatnitd) [reply src/components/Admin/ListDevices/index.js]: Also, the hyperlink color of View | Edit | Delete is different from location color.

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Pull request merged**